docs/sources/plugins/developing/development.md

```diff
@@ -96,13 +93,13 @@ export {
 
 There are three ways that you can start developing a Grafana plugin.
 
-1. Setup a Grafana development environment. [(described here)](http://docs.grafana.org/project/building_from_source/) and place your plugin in the ```data/plugins``` folder.
-2. Install Grafana and place your plugin in the plugins directory which is set in your [config file](/installation/configuration). By default this is `/var/lib/grafana/plugins` on Linux systems.
-3. Place your plugin directory anywhere you like and specify it grafana.ini.
+1. Setup a Grafana development environment [(described here)](http://docs.grafana.org/project/building_from_source/) and place your plugin in the ```data/plugins``` folder.
+2. Install Grafana and place your plugin in the plugins directory which is set in your [config file](/docs/grafana/latest/installation/configuration). By default, this is `/var/lib/grafana/plugins` on Linux systems.
+3. Place your plugin directory anywhere you like and specify it in grafana.ini.
 
-We encourage people to setup the full Grafana environment so that you can get inspiration from the rest of grafana code base.
+We encourage people to setup the full a Grafana environment so that you can get inspiration from the rest of grafana code base.
```

Comment:
```suggestion
We encourage people to set up the full Grafana environment so that you can get inspiration from the rest of the Grafana code base.
```

Reply:
Watch out for "setup." "Setup" is a noun. "Set up" is a verb.